src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -543,22 +543,8 @@ private static void SubstituteBidiMirrors(FontMetrics fontMetrics, GlyphSubstitu
                         continue;
                     }
 
-                    CodePoint codePoint = codePointEnumerator.Current;
```

Comment:
This fixes an issue discovered downstream where we were incorrectly trimming whitespace from the beginning of new lines following a mandatory line break.